### DISCUSSION TITLE
Why paywalled?

### DISCUSSION THREAD

**Discussion by alexavil** [community/community#19080] [Copilot Conversations]
After enjoying the Technical Preview, the extension just died and it now offers to pay.
$10/month sounds too much, especially for something that's learning on FOSS projects.
Why not just make it free for public repos? Why do we have to maintain a large project to get free access?

**Comment by rauzerfabiano** [comment]
Unfortunately we feed something to have you pay now. Sense? Zero

**Comment by NeuralAIM** [comment]
This is the cheapest feature among other similar tools. Despite the fact that there are no analogues in terms of quality. They spend their money to keep neural networks on their servers, and the price of video cards leaves much to be desired.

**Comment by ffxsam** [comment]
$10/mo isn't worth all the time Copilot saves you?

**Comment by secretgspot** [comment]
never trust someone like Microsoft that now wants you to have internet access and ms account in order to log in into your own computer and os that you paid for!
